.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: java
 jdk:
   - oraclejdk7
-  - openjdk6
```

Comment:
Dropping support for an environment should be called out explicitly rather than hidden in the patch.  We'll probably replace this with openjdk7.